### PR TITLE
Fixes #220 - Readme & demo need to show `Show-ObjectTree`

### LINKED ISSUE #220: Update readme & demo for `Show-ObjectTree`
### Prerequisites

- [X] Write a descriptive title.
- [X] Make sure you are able to repro it on the latest version
- [X] Search the existing issues.

### Steps to reproduce

n/a

### Expected behavior

```console
Readme & demo illustrate Show-ObjectView
```

### Actual behavior

```console
Readme & demo don't illustrate Show-ObjectView
```

### Error details

_No response_

### Environment data

```PowerShell
n/a
```

### Version

7.5

### Visuals

_No response_

### PR DESCRIPTION
**Pull request by tig**
Fixes: #220.

- No code changes, just the .gif and readmes.

### CODE REVIEW

File: src/Microsoft.PowerShell.ConsoleGuiTools/Microsoft.PowerShell.ConsoleGuiTools.psd1 [hunk 1]
```diff
@@ -227,3 +227,8 @@ Initial Release
 # DefaultCommandPrefix = ''
 
 }
+
```
Comment (SteveL-MSFT) [inline src/Microsoft.PowerShell.ConsoleGuiTools/Microsoft.PowerShell.ConsoleGuiTools.psd1:230]: please remove these extra newlines

File: demo.ps1 [hunk 2]
```diff
@@ -14,4 +14,7 @@ killp
 killp note
 # .Silent cls
 # Example 4: Navigate PowerShell command history (Map this to F7 with https://github.com/gui-cs/F7History)
-Get-History | Sort-Object -Descending -Property Id -Unique | Select-Object CommandLine -ExpandProperty CommandLine | Out-ConsoleGridView -OutputMode Single -Filter $line -Title "Command Line History"
\ No newline at end of file
+Get-History | Sort-Object -Descending -Property Id -Unique | Select-Object CommandLine -ExpandProperty CommandLine | Out-ConsoleGridView -OutputMode Single -Filter $line -Title "Command Line History"
+# .Silent cls
+#  Example 4: Use Show-ObjectTree to output processes to a tree view
+Get-Process | Show-ObjectTree
```
Comment (SteveL-MSFT) [inline demo.ps1:20]: add one newline at end of file

File: README.md [hunk 3]
```diff
@@ -124,6 +130,14 @@ Whatever you select within `Out-ConsoleGridView` will be inserted on your comman
 
 Whatever was typed on the command line prior to hitting `F7` or `Shift-F7` will be used as a filter.
 
+### Example 9: Output processes to a tree view
+
+```PowerShell
+PS C:\> Get-Process | Show-ObjectTree
+```
+
+This command gets the processes running on the local computer and sends them to a tree view window.
```
Comment (SteveL-MSFT) [inline README.md:139]: It may be worth stating ``Use right arrow where you see a `+` symbol to expand the tree``

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]

File: Build.ps1 [hunk 4]
```diff
@@ -26,6 +26,7 @@ if ($psd1Content) {
     }
     "Rewriting $PsdPath with new ModuleVersion: $ModuleVersion"
     $updatedpsd1Content = $psd1Content -replace "ModuleVersion\s+=\s+'([\d\.]+)'", "ModuleVersion = '$ModuleVersion'"
+    $updatedpsd1Content = $updatedpsd1Content.TrimEnd()
```
Comment (andyleejordan) [inline Build.ps1:29]: Love it, gotta automate the formatting!😄

**Review by andyleejordan — APPROVED** [review]
Thanks!!!

**Pull request merged**